src/Components/AdvancedSearch/AdvancedSearchModal.tsx

```diff
@@ -14,13 +14,35 @@ import {
 } from '@fluentui/react';
 import { useId } from '@fluentui/react-hooks';
 import QueryBuilder from './Internal/QueryBuilder';
-import AdvancedSearchResults from './Internal/AdvancedSearchResults';
+import AdvancedSearchResultDetailsList from './Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList';
+import { IADTTwin } from '../../Models/Constants';
+import twinData from '../../Adapters/__mockData__/MockAdapterData/DemoEnvsTwinData.json';
 
 const getClassNames = classNamesFunction<
     IAdvancedSearchStyleProps,
     IAdvancedSearchStyles
 >();
 
+const filteredTwins: IADTTwin[] = twinData;
+
+const cols = [
```

Comment:
let's just pass in the column names (FieldName) instead of the full column objects.